### PR TITLE
feat: able to set cell size for each location;add two ttfs

### PR DESCRIPTION
**Pull request by bjwswang**
Fix issue found at this PR https://github.com/bestchains/bc-cli/pull/50#issuecomment-1612373447

@dayuy @0xff-dev 

image

### CODE REVIEW

**Comment by bjwswang** [comment]
@0xff-dev  use `resource/ttf/SourceHanSansCN-Normal` instead of `stsong.ttf` in default deployment

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{
```
Comment (0xff-dev) [inline .vscode/launch.json:1]: Delete this file. Maybe we should add `.gitginore`.

**Comment by 0xff-dev** [comment]
> @0xff-dev use `resource/ttf/SourceHanSansCN-Normal` instead of `stsong.ttf` in default deployment

https://github.com/bestchains/bc-saas/blob/main/deploy/deploy.yaml#L39  It also needs to be updated here

File: cmd/depository/main.go [hunk 2]
```diff
@@ -52,7 +52,7 @@ var (
 
 	// flags for depository certificate generation
 	templateImagePath = flag.String("cert-template-image", "resource/certificate_template.jpg", "template image for depository's certificate generation")
-	ttfFontPath       = flag.String("cert-ttf-font", "resource/ttf/stsong.ttf", "ttf font file for depository's certificate generation")
+	ttfFontPath       = flag.String("cert-ttf-font", "resource/ttf/SourceHanSansCN-Normal.ttf", "ttf font file for depository's certificate generation")
```
Comment (0xff-dev) [inline cmd/depository/main.go:55]: The default path is `resource/ttf/SourceHanSansCN-Normal.ttf`, But in the image, our executable file is under `/bin` and the template files are under `/bc-saas/resource`, so the code should panic if it is run without the relevant parameters.

Comment (0xff-dev) [reply cmd/depository/main.go:55]: The executable files in the image should also be placed under `/bc-saas`.

Comment (bjwswang) [reply cmd/depository/main.go:55]: that's fine. Its deployment relevant.

File: .vscode/launch.json (hunk 1 above)
Comment (bjwswang) [reply .vscode/launch.json:1]: the `.vscode` is in `.gitignore`... 
anyway i remoted the vscode launch

**Review by 0xff-dev — APPROVED** [review]

**Review by Abirdcfly — APPROVED** [review]

**Pull request merged**